views/components/Header.jsx

```diff
@@ -1,35 +1,27 @@
 import React, { useContext } from 'react'
-import { useNavigate } from 'react-router-dom'
-import AppBar from '@material-ui/core/AppBar'
-import IconButton from '@material-ui/core/IconButton'
-import Toolbar from '@material-ui/core/Toolbar'
-import Typography from '@material-ui/core/Typography'
+import { useNavigate, useParams } from 'react-router-dom'
+import { AppBar, IconButton, Toolbar, Typography } from '@material-ui/core/'
 import ColorLens from '@material-ui/icons/ColorLens'
 import Person from '@material-ui/icons/Person'
-import openNewWindow from '../fe-utils/windowUtil'
 import { ThemeContext } from '../contexts/ThemeContext'
 import basePathConfig from '../../server/configs/basePathConfig'
 import { routes } from '../routes/routes'
 
 const basePath = basePathConfig || ''
 
-const Header = ({ handleDrawerToggle, isAccountPage, title }) => {
+const Header = ({ onClose, isAccountPage, title, onShow }) => {
   const { classes } = useContext(ThemeContext)
   const navigate = useNavigate()
+  const { study, subject } = useParams()
 
   return (
     <AppBar className={classes.appBar}>
-      <Toolbar style={{ paddingLeft: '16px' }}>
-        <IconButton
-          color="default"
-          aria-label="Open drawer"
-          onClick={handleDrawerToggle}
-          className={classes.navIconHide}
-        >
+      <Toolbar>
+        <IconButton color="default" aria-label="Open drawer" onClick={onClose}>
           <img width="24px" height="24px" src={`${basePath}/img/favicon.png`} />
         </IconButton>
         <Typography variant="title" color="inherit" className={classes.title}>
-          {title}
+          {onShow ? title : subject + ' - ' + study}
```

Comment:
we're currently passing the title in as a prop. Should we do this logic in the parent component and pass it as the title so we don't need to make this change?

Reply:
Thanks for this
[2c26c5e](https://github.com/AMP-SCZ/dpdash/pull/352/commits/2c26c5ef82ff63420783fb443c4cd7a14df00535)